src/openshiftCommonUI.module.js

```diff
@@ -5,6 +5,15 @@
  *   Base module for openshiftCommonUI.
  */
 angular.module('openshiftCommonUI', [])
+// Sometimes we need to know the css breakpoints, make sure to update this
+// if they ever change!
+.constant("BREAKPOINTS", {
+  screenXsMin:  480,   // screen-xs
+  screenSmMin:  768,   // screen-sm
+  screenMdMin:  992,   // screen-md
+  screenLgMin:  1200,  // screen-lg
+  screenXlgMin: 1600   // screen-xlg
+})
```

Comment:
Good call, @jeff-phillips-18.  Changed in https://github.com/openshift/origin-web-catalog/pull/261/commits/a08c2a4f12925a0f4a37cbbe8602cb794ff4a04b#diff-052c10f2f663033239b7d443fe45f0eaR52 (along with SaaS mobile styling changes).